db/column_family.cc

```diff
@@ -1293,6 +1293,13 @@ void ColumnFamilyData::InstallSuperVersion(
         mutable_cf_options.write_buffer_size) {
       mem_->UpdateWriteBufferSize(mutable_cf_options.write_buffer_size);
     }
+    if (old_superversion->mutable_cf_options.disable_auto_flush != mutable_cf_options.disable_auto_flush) {
+      assert(!mutable_cf_options.disable_auto_flush);
+      ROCKS_LOG_INFO(ioptions_.info_log, "Enabling auto flush for column family: %s",
+                     GetName().c_str());
+      mem_->EnableAutoFlush();
```

Comment:
Make sure (through code inspection) `mem_` will never be `nullptr`.

Reply:
yeah. `mem_` shouldn't be null when we call `InstallSuperVersion`.